lib/p2p/Peer.ts

```diff
@@ -794,22 +847,19 @@ class Peer extends EventEmitter {
    * Completes the handshake by sending the [[SessionAckPacket]] and our [[SessionInitPacket]] if it
    * has not been sent already, as is the case with inbound peers.
    */
-  private completeHandshake = async (ownNodeState: NodeState, nodeKey: NodeKey, sessionInit: packets.SessionInitPacket) => {
+  private completeHandshake = async (ownNodeState: NodeState, ownNodeKey: NodeKey, ownVersion: string, sessionInit: packets.SessionInitPacket) => {
     if (!this.inbound) {
       // outbound handshake
       await this.ackSession(sessionInit);
     } else {
       // inbound handshake
       await this.ackSession(sessionInit);
-      await this.initSession(ownNodeState, nodeKey, sessionInit.body!.nodeState.nodePubKey);
+      await this.initSession(ownNodeState, ownNodeKey, ownVersion, sessionInit.body!.nodePubKey);
     }
   }
 
   private checkPairs = () => {
-    const inactivePairs = this.advertisedPairs.filter(pair => !this.activePairs.has(pair));
-    if (inactivePairs.length) {
-      this.emit('pairsAdvertised', inactivePairs);
-    }
+    this.emit('verifyPairs');
```

Comment:
would it make sens to inline this on 279 since the function is now a 1-liner?

Reply:
Thought about doing that but left it like that in case `checkPairs` takes on more logic, but on some further thought I don't know when that would happen, so I went and made it an inline function like you suggested.